README.md

```diff
@@ -16,30 +16,61 @@
   * [Join the Gitter chat room](https://gitter.im/cocotb)
 
 ## Installation
+Cocotb can be installed by running `pip install cocotb`, and requires:
 
-Cocotb can be installed by running `pip install cocotb`.
+- Python 3.5+
+- A C++ compiler
+- An HDL simulator (such as [Icarus Verilog](http://iverilog.icarus.com/))
```

Comment:
Please turn things around here and put them in "action" style: first the requirements, then the installation.